keras_cv/bounding_box/utils.py

```diff
@@ -87,3 +87,44 @@ def _format_outputs(boxes, squeeze):
     if squeeze:
         return tf.squeeze(boxes, axis=0)
     return boxes
+
+
+def pad_with_sentinels(bounding_boxes):
+    # padded ragged bounding box tensor with -1s to create a tensor
+    # This enables vectorized map execution
+    return bounding_boxes.to_tensor(-1)
+
+
+def filter_sentinels(bounding_boxes):
```

Comment:
same here, docstring.  we should document what bounding_boxes is (a -1 padded tensor with axis 5 representing class), and document exactly what this does.

Reply:
done!